zzzz_modular_occulus/code/modules/reagents/reagents/other.dm

```diff
@@ -5,4 +5,247 @@
+
+/datum/reagent/drug/bliss
+	id = "bliss"
+	name = "Bliss"
+	description = "Looks as though it would metabolize into the ultimate hallucinogenic cocktail"
+	taste_description = "sweet"
+	color = "#ffc0eb"
+	metabolism = 0.25 * REM
+	var/init = 0
+	overdose = 15
+	addiction_chance = 60
+
+/datum/reagent/drug/bliss/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	if(!init)
+		to_chat(M, "<span class='warning'>Everything seems to turn glossy and wonderful.</span>")
+		init = 1
+	if(prob(5))
+		if(prob(98))
+			var/list/bliss_messages = list("You feel as if everything is great",
+			"The world seems bright and sparkly", "Everyone enjoys being with you.",
+			"You feel like theres nothing you can't do", "You feel a weight lifting off your chest",
+			"Nothing can go wrong for you",
+			"All your troubles seem to wash away")
+
+			to_chat(M, "<font color='#e3209b'>[pick(bliss_messages)]</font>")
+	sanity_gain = 4
+	M.druggy = max(M.druggy, 5 * effect_multiplier)
+	M.make_dizzy(10 * effect_multiplier)
+	return ..()
+
+/datum/reagent/frioline
+	id = "frioline"
+	name = "Frioline"
+	description = "Could cause rapid and sustained decrease in body temperature"
+	color = "#A0E1F7"
+
+/datum/reagent/frioline/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	if(M.bodytemperature > 310)
+		to_chat(M, "<span class='notice'>You suddenly feel very cold.</span>")
+	M.bodytemperature = max(165, M.bodytemperature - 30)
+	return ..()
+
+/datum/reagent/luxitol
+	id = "luxitol"
+	name = "Luxitol"
+	description = "Mimics compounds in known connection with bioluminescence"
+	color = "#61E34F"
+	metabolism = 0.4 * REM
+
+/datum/reagent/luxitol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	M.set_light(l_range = 4.5, l_power = 2.5, l_color = COLOR_YELLOW)
+	return ..()
+
+/datum/reagent/luxitol/on_mob_delete(var/mob/living/M)	//Needs testingx
+	if(istype(M))
+		M.set_light(0)
+	return ..()
+
+/datum/reagent/liquid_skin
+	id = "liquid_skin"
+	name = "Liquid Skin"
+	description = "Fills in microscopic ridges on biotic surfaces and hardens"
+	color = "#F7E9BE"
+
+/datum/reagent/liquid_skin/touch_mob(var/mob/M, var/volume)
+	if(istype(M, /mob/living))
+		var/mob/living/L = M
+		var/burned = L.getFireLoss() > 0
+		if(burned)
+			to_chat(usr, "In a strange sensation, you feel some burns stop hurting.")
+			L.heal_organ_damage(0, min(15, volume / 4))
+
+		if (mFingerprints in M.mutations)
+			if(!burned)
+				to_chat(usr, "Another application of the substance does nothing weird to your hands.")
+		else
+			M.mutations.Add(mFingerprints)
+			to_chat(usr, "Your fingers feel strange after the substance splashes on your hands.")
+	return ..()
+
+/datum/reagent/paralitol
+	id = "paralitol"
+	name = "Paralitol"
+	description = "Seems as if it could work as an extreme muscle inhibitor"
+	color = "#2F85CC"
+	metabolism = 3 * REM
+
+/datum/reagent/paralitol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	M.Weaken(12)
```

Comment:
This could be problematic. Chloral is already high-key kinda OP, this stuff is gonna be worse.